### ISSUE TITLE
No way to go back after choosing google login

### ISSUE THREAD

**Issue by fimdomeio** [RocketChat/Rocket.Chat#1463]
Minor interface issue.
I have google login activated in my rocket chat install and I have a regular account. If I inadvertly click the google login button I get to the register username window. From there I can't go back not even on page refresh. I have to register with google then logout then login with my regular account.
##

**Comment by timkinnane** [comment]
+1. Came to report this.
Tried to login (iOS app) with Twitter to authenticate because I thought that's how I registered the first time, but it must not have been because it went into the create account / choose username screen. I don't want to create another user, but I can't go back, there's no UI on this screen other than account name. I tried restarting the app, but it restores the same state.

https://www.dropbox.com/s/kfbxsvckdvtdicb/2016-05-20%2010.22.23.png?dl=0
